core/src/test/resources/accumulo2.properties

```diff
@@ -21,7 +21,6 @@ general.rpc.timeout=123s
 instance.secret=mysecret
 instance.volumes=hdfs://localhost:8020/accumulo123
 instance.zookeeper.host=myhost123:2181
-table.durability=flush
```

Comment:
This looks safe. I don't think any test code was relying on this being `flush` instead of the default, `sync`.

Reply:
I moved this out because some test code would attempt create a SiteConfig obj and that now fails.  I examined all the test code using this and none explicitly depended on this prop AFACIT.